README.md

```diff
@@ -16,32 +16,32 @@
 
 ## Executive summary
 
-- Immuni is a technological solution that centres on **an iOS and Android smartphone app.** It helps us fight the COVID-19 pandemic by notifying users at risk of carrying the virus as early as possible—**even when they are asymptomatic.** These users can then isolate themselves to avoid infecting others, and seek medical advice.
-- Immuni’s design and development are founded on five main principles: **utility, accuracy, scalability, transparency,** and **privacy.**
+- Immuni is a technological solution that centres on **an iOS and Android smartphone app.** It helps us to fight the COVID-19 pandemic by notifying users at risk of carrying the virus as early as possible—**even when they are asymptomatic.** These users should then isolate themselves to avoid infecting others, and seek medical advice.
+- Immuni’s design and development are based on five main principles: **utility, accuracy, scalability, transparency,** and **privacy.**
```

Comment:
Hi @ennioma! First of all, thanks for taking the time. We truly appreciate your contribution. Please note that all the copy has been reviewed with two native British English copywriters and our legal counsel. I state this to clarify that we'll need to be very considerate with every change.

Our copywriters would like to keep "can," rather than "should."

The use of "based" instead of "founded" is acceptable and we'll be happy to merge a PR with that change.

Reply:
Thanks for the answer, I was pretty sure this copy should be reviewed by legal counsel too. 
I think with the `can` you make the intention of your app weaker. 

Btw thanks for your effort publishing this (and I hope the code soon) on GitHub, really appreciated.

Reply:
We're doing our best! More is coming.